change/@fluentui-react-menu-e1cd8e82-e971-4a23-b76a-d95289214a1d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31396&commitId=0943fd71d73c267e4189b0e2bece0b684932b2f9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Menu Converged - groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - groups.default - Dark Mode.chromium.png|284| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|555| Changed|
|Menu Converged - groups.default.chromium.png|344| Changed|
|Menu Converged - groups.default - RTL.chromium.png|344| Changed|

</details>


  <details><summary><b>Menu Converged - selection groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection groups.default.selected.chromium.png|347| Changed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|343| Changed|
|Menu Converged - selection groups.default.chromium.png|347| Changed|
|Menu Converged - selection groups.default - RTL.chromium.png|343| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>